packages/better-php-doc-parser/src/PhpDocParser/ClassAnnotationMatcher.php

```diff
@@ -76,12 +76,12 @@ private function resolveFullyQualifiedClass(array $uses, Node $node, string $tag
     private function matchFullAnnotationClassWithUses(string $tag, array $uses): ?string
     {
         foreach ($uses as $use) {
-            foreach ($use->uses as $useUse) {
-                if (! $this->isUseMatchingName($tag, $useUse)) {
+            foreach ($use->uses as $use) {
```

Comment:
Here is foreached variable override, it should be skipped too.

After this foreach, the original `$use` might be overriden by iterated `$use`

Reply:
Fixed by skip property fetch not from `ThisType`